cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,52 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomeAndCataloguePageObject from
+  '../support/pages/homeСatalogue.pageObject';
```

Comment:
there is no such file

Reply:
> there is no such file

Sorry, but this file was already there and I didn't change it. 
You can find it here.
cypress/support/pages/homeCatalogue.pageObject